### PR TITLE
Updated dev dependencies

### PR DESCRIPTION
**Pull request by quartercastle**
- Updated dev dependencies 
- Updated eslint config to support the new version
- Fix some minor linting issues relating to the eslint update 
- Removed `'use strict'` from modules, its not necessary in ES6 and the code is linted and transpiled to ES5 anyway

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -56,7 +56,6 @@
     "no-caller": 2,
     "no-div-regex": 2,
     "no-else-return": 2,
-    "no-empty-label": 2,
```
Comment (quartercastle) [inline .eslintrc:59]: Rule `no-empty-label` is removed and replaced by `no-labels`

File: .eslintrc [hunk 2]
```diff
@@ -172,14 +175,13 @@
...
     ],
-    "space-after-keywords": 2,
+    "keyword-spacing": 2,
     "space-before-blocks": 2,
     "object-curly-spacing": 2,
     "array-bracket-spacing": 2,
     "computed-property-spacing": 2,
     "space-in-parens": 2,
     "space-infix-ops": 2,
-    "space-return-throw-case": 2,
```
Comment (quartercastle) [inline .eslintrc:182]: Replaced by `keyword-spacing`

File: .eslintrc [hunk 3]
```diff
@@ -172,14 +175,13 @@
         "ignoreCase": true
       }
     ],
-    "space-after-keywords": 2,
```
Comment (quartercastle) [inline .eslintrc:175]: Replaced by `keyword-spacing`

**Comment by hrmoller** [comment]
Hi @kvartborg

Thanks for your PR we'll have a look at it ASAP.

**Review by hrmoller — APPROVED** [review]
Changes approved. Will be released in a version 2.x when #2 is completed.

**Comment by hrmoller** [comment]
Released: https://www.npmjs.com/package/passport-unilogin

Thanks for your contributions @kvartborg

**Pull request merged**